package.json

```diff
@@ -27,13 +27,12 @@
     "babel-plugin-transform-strict-mode": "6.22.0",
     "babel-plugin-transform-class-properties": "6.23.0",
     "babel-polyfill": "6.23.0",
-    "babel-preset-es2015": "6.24.0",
+    "babel-preset-es2017": "6.22.0",
```

Comment:
No comment in a JSON unfortunately.
And nope, the 6.24 version of this package doesn't exist; still we move from `es2015` to `es2017` presets.

And the official language for this project is english, sorry :)